### PR TITLE
WD-290: Provide support for V4

### PR DESCRIPTION
**Pull request by ingalless**
Goal is to release as version 2.0. I'll update the docs as part of preparing for release :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
-  "dependencies": {
-    "@azure/identity": "^3.0.0",
-    "@microsoft/microsoft-graph-client": "^3.0.2",
-    "isomorphic-fetch": "^3.0.0"
+    "dist": "./dist"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "format:check": "prettier --check src/**/*.{ts,tsx}",
+    "format": "prettier --write src/**/*.{ts,tsx}",
```
Comment (dnlyates) [inline package.json:12]: Since we've got Prettier included it might be worth adding our config

File: src/index.ts
Comment (dnlyates) [inline src/index.ts:1]: Typescript ❤️

**Review by KobanDavis — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ingalless) [reply package.json:12]: Good spot, done

Comment (dnlyates) [reply package.json:12]: You need to also add `"prettier": "@mondago/prettier-config"` to package.json

**Review by dnlyates — APPROVED** [review]

**Pull request merged**